src/SafeDNS/Entities/Template.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace UKFast\SDK\SafeDNS\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property integer $id
+ * @property string $name
+ * @property boolean $default
+ * @property string $createdAt
```

Comment:
With dates in the SDK, these need to be `DateTime` objects.

You could do this by adding `createdAt` to the `$dates` property of this class, then you can sit back and have the SDK handle the transformation :sunglasses: